resources/sass/app.scss

```diff
@@ -24,8 +24,31 @@
 	width: 45%;
 }
 
+#ext-image-2{
+	height: 20%;
+	width: 100%;
+	background-color: green;
```

Comment:
Non-standard, debug skinning/styling should not be committed unless explicitly marked for removal when we create a release branch.